client/go/commands/flags_test.go

```diff
@@ -0,0 +1,88 @@
+//
+// Copyright (c) 2015 The heketi Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package commands
+
+import (
+	"bytes"
+	"github.com/gorilla/mux"
+	"github.com/heketi/heketi/apps/glusterfs"
+	"github.com/heketi/heketi/tests"
+	"net/http/httptest"
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestJsonFlagsCreate(t *testing.T) {
+	defer os.Remove("heketi.db")
+
+	// Create the app
+	app := glusterfs.NewApp()
+	defer app.Close()
+	router := mux.NewRouter()
+	app.SetRoutes(router)
+
+	// Setup the server
+	ts := httptest.NewServer(router)
+	defer ts.Close()
+
+	//set options
+	options := &Options{
+		Url:  ts.URL,
+		Json: true,
+	}
+
+	//create b to get values of stdout
+	var b bytes.Buffer
+	defer tests.Patch(&stdout, &b).Restore()
+
+	//create mock cluster and mock destroy
+	mockCluster := NewCreateNewClusterCommand(options)
+
+	//create new cluster and assert json
+	err := mockCluster.Exec([]string{})
+	tests.Assert(t, err == nil)
+	tests.Assert(t, strings.Contains(b.String(), "{\"id\":\""), b.String())
```

Comment:
Just a suggestion.. instead of using `\"`, you can just change the entire string to just the tick mark and double quotes will not be interpreted.  See app_X_test.go in glusterfs for examples.